projection/ibc_channel/view/ibc_channels.go

```diff
@@ -450,12 +451,26 @@ func (ibcChannelsView *IBCChannels) List(order IBCChannelsListOrder, pagination
 		"bonded_tokens",
 	).From(
 		"view_ibc_channels",
+	).Where(
+		"status = ?", "true", // Only if status is true, the channel is in `opened` status
```

Comment:
Shall we make this a filter option?

Reply:
Sure! Let me make it enable by default.

Some extra context here. 

At the moment, as we did not handle ChannelClose and ChannelCloseOnTimeout (not sure if I write their name correctly here 😂 ), `status` field only indicated that the channel is successfully established, that is when `OpenChannelAck` or `OpenChannelConfirm` is detected, `status` will be set as `true`.

In the future, when we handle channel close properly, this field could really represent that the channel is `closed`.